roles/cis/tasks/6/6.1.10.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: "6.1.10 Ensure no world writable files exist (Scored)"
+  shell: |
```

Comment:
@satya-rajesh we should reduce shell and use it when absolutely necessary. let us try and find if we can use ansible modules

Reply:
Hi Priyesh,

In this case, the CIS rule mentions to execute the below to check it does not return anything:
$(df -P | awk '{if (NR!=1) print $6}' | xargs -I '{}' find '{}' -xdev -type f -perm -0002)

In my opinion it could be little tough to program in ansible, could you please advise.